### DISCUSSION TITLE
Sherten the CLI app command name to `th`

### DISCUSSION THREAD

**Discussion by ottobolyos** [TrakHound/TrakHound#4] [Ideas]
I think it might be a good idea to shorten the CLI command name to `th`. It reminds me of GitHub’s CLI client called [`gh`](https://cli.github.com/).

**Comment by PatrickRitchie** [comment]
Good suggestion! I am working on the next version [v0.1.4](https://github.com/TrakHound/TrakHound/tree/v0.1.4) so I will include this in that release.

Let me know if you have any other suggestions!
